CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to ActivityPub for WordPress
+
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+Developers of all levels can help — whether you can barely recognize a filter (or don’t know what that means) or you’ve already authored your own plugins, there are ways for you to pitch in.
+
+## Create Bug Reports
+
+If you find a bug, please [file a GitHub issue](https://github.com/Automattic/wordpress-activitypub/issues/). If you want to prefix the title with a “Question:”, “Bug:”, or the general area of the application, that would be helpful, but by no means mandatory. If you have write access, add the appropriate labels.
```

Comment:
I don't think that part is really necessary, since there are issue templates that can take care of that.

```suggestion
If you find a bug, please [file a GitHub issue](https://github.com/Automattic/wordpress-activitypub/issues/). If you have write access, add the appropriate labels.
```